README.md

```diff
@@ -5,11 +5,92 @@ Project to automatically convert news / entertainment websites into STAMP
 
 This is an internship project for Summer 2020.
 
-To get the Project working you will need to set the API key as an environment 
-variable with the name `GOOGLE_CLOUD_API_KEY` and the value as the base64 
-encoded string of the API key obtained from the console.
+## Project Requirements:
 
-Refer to this link for more information on creating and setting API keys:
-https://cloud.google.com/docs/authentication/api-keys
+- To get the project working, you will need to set the below mentioned API keys as environment 
+variables:
 
-To get the API key used for this project contact the owner of this repo.
+    -  `GOOGLE_CLOUD_API_KEY` and the value as the base64 encoded string of the API key obtained from the console.
+
+        Refer to this link for more information on creating and setting API keys:
+        https://cloud.google.com/docs/authentication/api-keys
+    
+        To get the API key used for this project contact the owner of this repo.
+    
+    -  `FLASK_APP_SECRET_KEY` and the value could be any random generated string.
+
+        Refer to this link for more information on the usage of flask secret key:
+        https://flask.palletsprojects.com/en/1.1.x/quickstart/#sessions
```

Comment:
This should be under
### Set environment variables

Reply:
Will be considered in coming iterations. Thanks for suggestion!

Reply:
Same for all other suggestions.